openapi.yml

```
@@ -220,6 +235,8 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/UserProfile"
+        404:
```

afaik status codes should be string, see https://swagger.io/docs/specification/describing-responses/

```suggestion
        "404":
```